app/javascript/gobierto_data/webapp/components/commons/InfoList.vue

```diff
@@ -1,24 +1,140 @@
+              {{ item.attributes.data_updated_at | convertDate }}
+            </span>
+          </div>
+          <div
+            v-if="item.attributes.frequency[0].name_translations"
+            class="gobierto-data-summary-header-container"
+          >
+            <i
+              class="fas fa-calendar"
+              style="color: var(--color-base); opacity: .25"
+            />
+            <span class="gobierto-data-summary-header-container-label">
+              {{ labelFrequency }}
+            </span>
+            <template v-if="selectedLanguage === 'es' && item.attributes.frequency[0].name_translations !== undefined">
+              <span class="gobierto-data-summary-header-container-text">
+                {{ item.attributes.frequency[0].name_translations.es }}
+              </span>
+            </template>
+            <template v-else-if="item.attributes.frequency[0].name_translations !== undefined">
+              <span class="gobierto-data-summary-header-container-text">
+                {{ item.attributes.frequency[0].name_translations.ca }}
+              </span>
+            </template>
+          </div>
+
+          <div
+            v-if="item.attributes.category[0].name_translations"
+            class="gobierto-data-summary-header-container"
+          >
+            <i
+              class="fas fa-tag"
+              style="color: var(--color-base); opacity: .25"
+            />
+            <span class="gobierto-data-summary-header-container-label">
+              {{ labelSubject }}
+            </span>
+            <template v-if="selectedLanguage === 'es' && item.attributes.category[0].name_translations !== undefined">
+              <a
+                href=""
+                class="gobierto-data-summary-header-container-text-link"
+              >
+                {{ item.attributes.category[0].name_translations.es }}
+              </a>
+            </template>
+            <template v-else-if="item.attributes.category.name_translations !== undefined">
+              <a
+                href=""
+                class="gobierto-data-summary-header-container-text-link"
+              >
+                {{ item.attributes.category.name_translations.ca }}
+              </a>
+            </template>
+          </div>
+        </div>
+        <div class="pure-u-1-2">
+          <p class="gobierto-data-summary-header-description">
+            {{ item.attributes.description }}
+          </p>
+        </div>
+      </div>
+    </div>
   </div>
 </template>
 <script>
-import InfoListElement from "./InfoListElement.vue";
 export default {
-  name: "Info",
-  components: {
-    InfoListElement
+  name: "InfoList",
+  filters: {
+    convertDate: function(valueDate) {
+      const newDateFromApi = new Date(valueDate)
+      return newDateFromApi.toLocaleDateString('es-ES', {
+          day : 'numeric',
+          month : 'short',
+          year : 'numeric'
+      })
+    }
```

Comment:
Awful. You're forcing spanish language. We've a `date` filter (and a method) in [vue-filters.js](https://github.com/PopulateTools/gobierto/blob/master/app/javascript/lib/shared/modules/vue-filters.js)

Import it here. Perhaps do you need to modify the options? Sure:
```js
import { date } from '......../vue-filters.js'

filters: {
  convertDate(valueDate) {
        return date(valueDate, { your_awesome_options })
  }
}
```

Reply:
Thanks. I didn't know it.